lib/jellyfish_web/peer_socket.ex

```diff
@@ -130,14 +130,10 @@ defmodule JellyfishWeb.PeerSocket do
   @impl true
   def terminate(_reason, state) do
```

Comment:
I would get rid of this function at all as it won't be called in every case so sometimes we get this log and soimetimes not

Reply:
This function is required by behavior `Phoenix.Socket.Transport`, but if you want I can remove this log from it.

Reply:
Yeah, let's do this